test/toolkits/style_transfer/utils.hpp

```diff
@@ -24,8 +24,8 @@ std::string generate_data(size_t data_size);
 turi::flex_image random_image();
 turi::gl_sarray random_image_sarray(size_t length);
 turi::gl_sframe random_sframe(size_t length, std::string image_column_name);
-std::string download_ml_model(std::string download_url,
-                              std::string download_directory);
+std::string get_vgg16_model();
+std::string get_resnet_model();
```

Comment:
Is this resnet-16? resnet-50? Some other resent?

Reply:
resent_16. I can name it more specifically to reduce confusion.